classes/locations.php

```diff
@@ -248,28 +251,28 @@ function geo_location( $post ) {
 
 					<!-- City / Town -->
 					<tr valign="top">
-						<td><label for="location_city"><?php _e( 'City/Town', 'SimpleMap' ); ?></label></td>
+						<td><label for="location_city"><?php _e( 'City/Town', 'simplemap' ); ?></label></td>
 						<td><input type="text" name="location_city" id="location_city"
 								   value="<?php echo esc_attr( $location_city ); ?>" size="30"/></td>
 					</tr>
 
 					<!-- State / Providence -->
 					<tr valign="top">
-						<td><label for="location_state"><?php _e( 'State/Province', 'SimpleMap' ); ?></label></td>
+						<td><label for="location_state"><?php _e( 'State/Province', 'simplemap' ); ?></label></td>
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->